app/src/main/res/layout/activity_view_profile.xml

```diff
@@ -20,6 +20,7 @@
         app:popupTheme="@style/CustomPopupStyle">
 
         <TextView
+            android:id="@+id/textView_username"
```

Comment:
please use the activity name or identifier in the id. For example view_profile_username is much better for when you have to find id's and so that other people don't get confused as well. This applies to the other id on the same page